gradoop-temporal/src/main/java/org/gradoop/temporal/model/impl/functions/predicates/AsOf.java

```diff
@@ -32,14 +36,24 @@
   private final long queryTimestamp;
 
   /**
-   * Creates a AsOf instance with the given time-stamp.
+   * Creates an AsOf instance with the given time-stamp.
    *
    * @param timestamp The timestamp to match in Milliseconds since Unix Epoch.
    */
   public AsOf(long timestamp) {
     this.queryTimestamp = timestamp;
   }
 
+  /**
+   * Creates an AsOf instance with the given time-stamp.
```

Comment:
I think you should highlight the `AsOf` part somehow, either with `<b>` (to keep it consistent with the existing predicates) or with `@code`